### DISCUSSION TITLE
Directly require or remove Laravel Collections

### DISCUSSION THREAD

**Discussion by abenerd** [spatie/ignition#1] [Ideas]
Currently ignition is using the Laravel `Illuminate\Support\Collection` in several places. This class is only available transitively via the `spatie/flare-client-php` package that requires `illuminate/pipeline` which in turn requires `illuminate/collections`. I propose that if the Laravel Collections are going to be used to directly require the `illuminate/collections` package. Although I think the package doesn't even need the collections and the native php array functions would suffice.

**Comment by freekmurze** [comment]
Feel free to send a PR for this!
